diff -urN zvbi-0.2.2.orig/configure.in zvbi-0.2.2/configure.in
--- zvbi-0.2.2.orig/configure.in	Fri Oct 11 18:33:20 2002
+++ zvbi-0.2.2/configure.in	Fri Oct 11 18:38:26 2002
@@ -4,7 +4,6 @@
 AC_INIT(configure.in)
 AM_INIT_AUTOMAKE(zvbi, 0.2.2)
 AM_CONFIG_HEADER(config.h)
-AM_ACLOCAL_INCLUDE(m4)
 
 AC_PROG_CC
 AC_PROG_CXX dnl just for test
@@ -67,7 +66,7 @@
 dnl
 PNG_LIB=""
 HAVE_PNG="yes"
-AC_CHECK_LIB(png, png_destroy_write_struct, PNG_LIB="-lpng -lz -lm", HAVE_PNG="no", -lz -lm)
+AC_CHECK_LIB(png, png_destroy_write_struct, [PNG_LIB="-lpng"], [HAVE_PNG="no"])
 if test "x$HAVE_PNG" = "xyes"; then
   AC_DEFINE(HAVE_LIBPNG, 1, [Define if you have libpng])
 fi
