--- zvbi-0.1.1/configure.in~	Thu Apr  4 03:36:44 2002
+++ zvbi-0.1.1/configure.in	Thu Apr  4 03:39:28 2002
@@ -4,13 +4,6 @@
 AC_INIT(configure.in)
 AM_INIT_AUTOMAKE(zvbi, 0.1.1)
 AM_CONFIG_HEADER(config.h)
-AM_ACLOCAL_INCLUDE(macros)
-
-AC_ARG_ENABLE(debugging, [  --enable-debugging      add debugging info],
-        [CFLAGS="-g -O0"
-	 AC_DEFINE(ENABLE_DEBUGGING, 1, [Add debugging info])],
-        CFLAGS="$CFLAGS -O2")
-dnl -fomit-frame-pointer
 
 AC_PROG_CC
 AM_PROG_LIBTOOL
@@ -23,6 +16,7 @@
 #endif /* SITE_DEF_H */
 EOF
 
+AM_CONDITIONAL(ENABLE_GTK_DOC,0)
 dnl
 dnl Check whether program_invocation_name exists, GNU extension.
 dnl
@@ -56,7 +50,7 @@
 dnl
 PNG_LIB=""
 HAVE_PNG="yes"
-AC_CHECK_LIB(png, png_destroy_write_struct, PNG_LIB="-lpng -lz -lm", HAVE_PNG="no", -lz -lm)
+AC_CHECK_LIB(png, png_destroy_write_struct, [PNG_LIB="-lpng"], [HAVE_PNG="no"])
 if test "x$HAVE_PNG" = "xyes"; then
 AC_DEFINE(HAVE_LIBPNG, 1, [Define if you have libpng])
 fi
@@ -93,19 +87,10 @@
 dnl Native language support
 dnl
 ALL_LINGUAS="de es fr nl pl sv"
-AM_GNU_GETTEXT(, [need-ngettext])
-
-saved_ldflags="$LDFLAGS"
-if test "x$USE_INCLUDED_LIBINTL" = "xyes"; then
-LDFLAGS="$LDFLAGS intl/libintl.a"
-else
-LDFLAGS="$LDFLAGS $INTLLIBS"
-fi
+AM_GNU_GETTEXT
 
 AC_CHECK_FUNCS(ngettext)
 
-LDFLAGS="$saved_ldflags"
-
 dnl
 dnl Check for gtk-doc
 dnl
@@ -119,14 +104,7 @@
 src/Makefile
 intl/Makefile
 po/Makefile.in
-po/Makefile
 test/Makefile
 zvbi.spec
 doc/Makefile
 ])
-
-if test ! -r po/Makefile; then
-	PF="POT"
-	PF=$PF"FILES"
-        sed -e "/$PF =/r po/PF" po/Makefile.in > po/Makefile
-fi
